logstash-integration-elastic_enterprise_search.gemspec

```diff
@@ -0,0 +1,31 @@
+Gem::Specification.new do |s|
+  s.name          = 'logstash-integration-elastic_enterprise_search'
+  s.version         = '2.0.0'
```

Comment:
first version will be 2.0.0 than due logstash-output-elastic_app_search's latest release being 1.2.0, right?

Reply:
Correct, it's the same versioning strategy was used for Kafka Integration